ex.js

```diff
@@ -0,0 +1,43 @@
+// EXERCÍCIO 1
+console.log(`\nEXERCÍCIO 1:\n`);
+
+let hacker1 = `Marco Modena`;
+let hacker2 = `Vinicius Adamy`;
+
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
+
+// EXRCÍCIO 2
+console.log(`\nEXERCÍCIO 2:\n`);
+
+if(hacker1.length==hacker2.length){
+    console.log(`Wow, you both got equally long names, ${hacker1.split(" ").join("").length} characters!`);
+} else if (hacker1.length<hacker2.length) {
+    console.log(`Yo, navigator got the longest name, it has ${hacker2.split(" ").join("").length} characters`);
+} else {
+    console.log(`The Driver has the longest name, it has ${hacker1.length} characters`);
+}
+
+// EXERCÍCIO 3
+console.log(`\nEXERCÍCIO 3:\n`);
+
+console.log(hacker1.toUpperCase().split(" ").join("").split("").join(" "));
+console.log(hacker2.split("").reverse().join(""));
```

Comment:
Cuidado com o excesso de Loops, "toUpperCase", "split", "reverse" e "join", são todos loops, daria pra resolver isso com um "FOR" e "IFs", cuidado com a performace do código!